docs/api/commands/fixture.mdx

```diff
@@ -73,7 +73,8 @@ Pass in an options object to change the default behavior of `cy.fixture()`.
 
 - `cy.fixture()` yields the contents of the file. Formatting is determined by
   its file extension.
-- The yielded subject is **not** updated if the contents change on disk.
+- `cy.fixture()` is not a query. It
```

Comment:
Based on our definition of a query:
> Most commands are not retried because they could potentially change the state of the application under test
theres really no reason .fixture() isn't query command. It's "querying" the file for data

Reply:
I'd like to leave the language somewhat standardized. How about 

> `cy.fixture()` is not a query. It [only reads the data once](/guides/core-concepts/retry-ability#Only-queries-are-retried), If you want a command that re-reads the file until chained assertions pass, use [`cy.readFile()`](/api/commands/readfile) instead.